docs/examples/general/data_loading/index.py

```diff
@@ -0,0 +1,22 @@
+doc(title=("Data Loading", "="),
+    options=[":maxdepth: 2"],
+    entries=[
+        example_entry("external_input.ipynb", [
```

Comment:
Can we auto-generate these cross-references during docs generation? Like, grep our examples for every operator?